### ISSUE TITLE
In-notification replies causing duplications?

### ISSUE THREAD

**Issue by jhoff80** [ThomasGaubert/zephyr#56]
I'll have to experiment with this one more, I don't have much time until the weekend, but I use the Google Android Messages app for SMS/MMS.  I got a notification of a text and it showed up perfectly in Zephyr.  I then responded on my phone from the notification, without opening the app.  When the Android Messages app sends my message, it then adds my message to the same notification, and stays in place until I dismiss it.  Zephyr ended up duplicating the notification after I did this.

I think giving an exact example might help explain this better:

Friend -> Lol omg
Zephyr notification: "Friend:  Lol omg"
My reply on phone -> I know, right?
Zephyr notification: "Friend: Lol omg"

But yeah, I'll have to test some more this weekend to determine how repeatable this is or anything like that.

**Comment by ThomasGaubert** [comment]
Thanks for the report! I'd imagine this is due to a notification with the same ID being published by the Messages app (essentially updating it instead of creating a new one in the notification tray). I'll make sure Zephyr handles this correctly.

**Comment by jhoff80** [comment]
I'm unfortunately still seeing some weirdness with this specific app (Google's "Messages" SMS app).  I was using Zephyr for about 2 hours today.   First I received a single text message from one friend, and that message came through twice in Zephyr.  I then responded, and then it came through a third time in Zephyr.  Later on, another friend texted, and it came through only the once, but then repeated when I responded again as well.
